### DISCUSSION TITLE
Etherscan super slow? Target block in 5 hours?

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1296] [Q&A]
Is it just me, or is Etherscan-Rinkeby currently super slow... Maybe some Russian servers were cut off?
if you are looking for alternatives: https://medium.com/coinmonks/whats-the-best-ethereum-block-explorer-10-alternatives-to-etherscan-8eb5f2be263d
![image](https://user-images.githubusercontent.com/32838961/159037978-4afa1439-bb60-4c4e-a489-f6db3cbdc90b.png)

Under AnyBlock I can see, the block is already confirmed 129 times:
https://explorer.anyblock.tools/ethereum/ethereum/rinkeby/transaction/0x0875d07b0f8a77f12a0f552928360140eebd64afdda7ed70050bdf75515fd9c7

**Comment by cromewar** [comment]
Nice tip I experienced myself how etherscan velocity had decreased significantly.
